Ruby generator doesn't compile

When compiling from latest upstream/master:

make clean
make

[HOSTCXX] Compiling src/compiler/ruby_generator.cc
In file included from src/compiler/ruby_generator.cc:41:0:
./src/compiler/ruby_generator_map-inl.h: In function ‘std::mapstd::basic_string<char, std::basic_string<char> > grpc_ruby_generator::ListToDict(const std::initializer_liststd::basic_string<char >&)’:
./src/compiler/ruby_generator_map-inl.h:54:5: error: ‘cerr’ is not a member of ‘std’
     std::cerr << "Not every 'key' has a value in `values`."
     ^
make: **\* [objs/opt/src/compiler/ruby_generator.o] Error 1

--- a/src/compiler/ruby_generator_map-inl.h
+++ b/src/compiler/ruby_generator_map-inl.h
@@ -34,6 +34,7 @@
 #ifndef NET_GRPC_COMPILER_RUBY_GENERATOR_MAP_INL_H_
 #define NET_GRPC_COMPILER_RUBY_GENERATOR_MAP_INL_H_
 
+#include <iostream>
 #include <initializer_list>
 #include <map>
 #include <ostream>  // NOLINT

